keywords/LiteServNetMsft.py

```diff
@@ -177,7 +179,7 @@ def stop(self):
         Stops a .NET listener on a remote windows machine via ansible and pulls logs.
         """
 
-        binary_path = "couchbase-lite-net-msft-{}-liteserv/LiteServ.exe".format(self.version_build)
+        binary_path = "couchbase-lite-net-msft-{}-liteserv/net45/LiteServ.exe".format(self.version_build)
```

Comment:
We might wants to use keywords.utils.has_dot_net4_dot_5 to figure out whether or not net45 should be there in the path as inline 125 above